src/main/webapp/dev/js/common/checkBrowserVersion.js

```diff
@@ -1,99 +1,28 @@
-// Browser Compatibility and support
-const MSIE = 'Microsoft Internet Explorer';
-const MSIE_LOWEST_VERSION = 9;
-const CHROME = 'Chrome';
-const CHROME_LOWEST_VERSION = 15;
-const FIREFOX = 'Firefox';
-const FIREFOX_LOWEST_VERSION = 12;
-const SAFARI = 'Safari';
-const SAFARI_LOWEST_VERSION = 4;
+/* global UAParser:true */
 
 /**
- * Function to check browser version and alert if browser version is lower than supported
- * Adapted from http://www.javascripter.net/faq/browsern.htm
- *
+ * Function to check whether user's browser is in the
```

Comment:
Start the documentation with the third person declarative tense: e.g. `Checks if the user's...`